[draft] Case studies (#46416)
mui/material-ui: docs/pages/customers/moz.md

  </span>
</blockquote>
</span>
<span class="only-dark-mode">
<blockquote style="margin: 32px 0; padding: 24px 32px; background: var(--muidocs-palette-background-default); border-left: 6px solid var(--muidocs-palette-primary-main); color: #f5f5f7; border-radius: 8px; font-size: 1.15rem; font-style: italic;">
  MUI X provides access to a very optimized and consistent DX and UX for a key portion of our app.
  <br>
  <span style="display: block; margin-top: 12px; font-size: 1rem; font-style: normal; color: #bbb; font-weight: 500;">
    – Darko Cejkov, Front-end Developer
  </span>
</blockquote>
</span>

## Challenge

Moz needed a UI framework that could:

- Efficiently cater to customer requests and offer a robust handling of large keyword datasets.
- Support customization for filtering, sorting, grouping and showing data in complex data grids in a readable way.
- Give a consistent and standard experience to the UI across all user facing applications.

## Solution

To meet these challenges, Moz integrated **MUI X Premium**, leveraging its **Data Grid** and **Date Picker** components to create an optimized, interactive, and performant UI.
Key benefits of using MUI X included:

> - A great DX and UX. A developer-friendly and intuitive user experience.

Reviewer: ```suggestion
- Great DX and UX. A developer-friendly and intuitive user experience.
```

- Advanced customization capabilities. Data Grid API refs allowed them to build a UI for filtering and column management,a game-changer for their keyword analysis platform.
- Faster feature development accelerated product delivery across the platform.

## Results

> By adopting MUI X, Moz could handling large keyword datasets efficiently with MUI X's performance optimizations. It helped them scale their platform. They could neatly display vast amounts of keyword data that the end users found easy to navigate. The highly customisable data grid enabled them to offer advanced filtering and column management features to their end users.

Reviewer 2: ```suggestion
By adopting MUI X, Moz could handling large keyword datasets efficiently with MUI X's performance optimizations. It helped them scale their platform. They could neatly display vast amounts of keyword data that the end users found easy to navigate. The highly customizable Data Grid enabled them to offer advanced filtering and column management features to their end users.
```

> By adopting MUI X, Moz could handling large keyword datasets efficiently with MUI X's performance optimizations. It helped them scale their platform. They could neatly display vast amounts of keyword data that the end users found easy to navigate. The highly customisable data grid enabled them to offer advanced filtering and column management features to their end users.

Reviewer: ```suggestion
By adopting MUI X, Moz could handling large keyword datasets efficiently with MUI X's performance optimizations, which helped them scale their platform. They could neatly display vast amounts of keyword data that end users find easy to navigate. The highly customisable data grid enables them to offer advanced filtering and column management features to their end users.
```

## Developer experience

Moz developers found MUI documentation intuitive and in-depth, making customizations straightforward.

:::info

> Pro tip: "Read the docs thoroughly—there are many small examples that can greatly help with customization."

Reviewer: ```suggestion
Moz pro tip: "Read the docs thoroughly—there are many small examples that can greatly help with customization."
```
